skypy/power_spectrum/tests/test_camb.py

```diff
@@ -24,15 +26,17 @@ def test_camb():
     '''
     from skypy.power_spectrum import camb
 
+    Pl15massless = Planck15.clone(name='Planck 15 massless neutrino', m_nu=[0., 0., 0.]*units.eV)
+
     # test shape and compare with the mocked power spectrum
     redshift = [0.0, 1.0]
     wavenumber = np.logspace(-4.0, np.log10(2.0), 200)
-    pkz = camb(wavenumber, redshift, Planck15, 2.e-9, 0.965)
-    assert pkz.shape == (len(redshift), len(wavenumber))
-    assert allclose(pkz, test_pkz, rtol=1.e-4)
+    pzk = camb(wavenumber, redshift, Pl15massless, 2.e-9, 0.965)
+    assert pzk.shape == (len(redshift), len(wavenumber))
+    assert allclose(pzk, test_pzk, rtol=1.e-4)
```

Comment:
```suggestion
    pzk = camb(wavenumber, redshift, Planck15, 2.e-9, 0.965)
    assert pzk.shape == (len(redshift), len(wavenumber))
    assert allclose(pzk, test_pzk, rtol=1.e-4)
```
Revert to using Planck15 in the camb tests. You will still need to update camb_results.txt